Source/Fuse.Android/AndroidInterop.uno

```diff
@@ -29,6 +29,24 @@ namespace Fuse.Android.Bindings
 			return pendingIntent;
 		@}
 
+		[Foreign(Language.Java)]
+		public static extern(Android) Java.Object LaunchApp(string action, string applicationId)
+		@{
+			Intent pendingIntent = null;
+			Activity a = com.fuse.Activity.getRootActivity();
+
+			try
+			{
+				pendingIntent = a.getPackageManager().getLaunchIntentForPackage(applicationId);
+				a.startActivity(pendingIntent);
+			} catch (Exception e) {
```

Comment:
Consistency: Either put all braces on their own lines, or move the first brace to become `try {`.